packages/node-app/src/controllers/giveaway.controller.ts

```diff
@@ -45,6 +33,7 @@ export const listGiveaways = async (req: Request, res: Response) => {
       ...omit(giveaway, ['participants']),
       winners: giveawayWinners(giveaway),
       stats: giveawayStats(giveaway),
+      status: giveawayStatus(giveaway),
```

Comment:
Since we only need to know if the giveaway is invalid of not, this can be just a `isInvalid` boolean